api/v1beta1/openstackbaremetalset_types.go

```diff
@@ -36,10 +50,20 @@ type OpenStackBaremetalSetSpec struct {
 	// +kubebuilder:default={}
 	// +kubebuilder:validation:Optional
 	// BaremetalHosts - Map of hostname to control plane IP address for all nodes to provision
```

Comment:
Should probably change the comment to something like "Map of hostname to Instance Spec"